### DISCUSSION TITLE
Уточнить сообщение, отправляемое при использовании выключенной команды

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#10] [Предложения]
Если использовать выключенную в панели управления команду, то Лакуна отправит сообщение: "У вас недостаточно прав на выполнение данной команды". Считаю его немного не интуитивным, так как есть ограничения именно по правам: переопределения в дискорде, в случае с командами модерации и активностей - также переопределения в панели. Было бы лучше для таких случаев отправлять сообщение "Данная команда выключена в настройках сервера", либо, если возможно, скрывать её из списка команд. Так пользователю будет понятнее причина, почему команда не выполняется
![image](https://github.com/LacunaHub/.github/assets/121901513/d25ca6a6-1134-4cff-afc1-60eef80b16a7)
![image](https://github.com/LacunaHub/.github/assets/121901513/15694281-a6d8-4d6d-91bb-e26d11800e5a)

**Comment by danliyev** [comment]
Waiting for the next release
